oio/api/ec.py

```diff
@@ -206,7 +208,7 @@ class ECStream(object):
     Handles the different readers.
     """
     def __init__(self, storage_method, readers, range_infos, meta_length,
-                 fragment_length, reqid=None, perfdata=None):
+                 fragment_length, reqid=None, perfdata=None, logger_=None):
```

Comment:
This is confusing. I suggest renaming the global logger with upper-case characters, and keep `logger` for the keyword argument.